itest/metrics/listen.py

```diff
@@ -0,0 +1,21 @@
+import socket
+
+
+def listen():
+    # Create a TCP/IP socket
+    sock = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
+
+    # Bind the socket to the port
+    server_address = ('0.0.0.0', 1234)
+    sock.bind(server_address)
+
+    with open('/var/log/metrics/metrics.log', 'w') as fp:
+        while True:
+            data = sock.recv(4096)
+            print(data.decode('utf-8'))
```

Comment:
Just to be sure, this is intentional, right?

Reply:
if you look at `itest/docker-compose.yml`, you’ll see I’ve added a new “metrics” entry with static ip `10.5.0.7`